qiskit/aer/aer_provider.ipynb

```diff
@@ -187,13 +187,14 @@
    "source": [
     "### Starting simulation with a custom initial state\n",
     "\n",
-    "The `QasmSimulator` allows setting a custom initial statevector for the simulation. This means that all experiments in a Qobj will be executed starting in a state $|\\psi\\rangle$ rather than the all zero state $|0,0,..0\\rangle$. The custom state may be set using the `backend_options` keyword argument for `execute`, or the Aer backend `run` method.\n",
+    "The `QasmSimulator` allows setting a custom initial statevector for the simulation. This means that all experiments in a Qobj will be executed starting in a state $|\\psi\\rangle$ rather than the all zero state $|0,0,..0\\rangle$. The custom state may be set in the circuit using the `initialize` method.\n",
     "\n",
     "**Note:**\n",
     "* The initial statevector must be a valid quantum state $|\\langle\\psi|\\psi\\rangle|=1$. If not an exception will be raised. \n",
     "* If a Qobj contains multiple circuits, the initial statevector must be the correct size for *all* experiments in the Qobj, otherwise an exception will be raised.\n",
```

Comment:
This bullet point can be removed since it doesn't apply anymore since the initialize is specified in each circuit rather than as a qobj config for all circuits.